app/services/claim_fast_tracking/max_rating_annotator.rb

```diff
@@ -1,11 +1,32 @@
 # frozen_string_literal: true
 
+require 'virtual_regional_office/client'
+
 module ClaimFastTracking
   class MaxRatingAnnotator
     def self.annotate_disabilities(rated_disabilities_response)
-      tinnitus = ClaimFastTracking::DiagnosticCodes::TINNITUS
-      rated_disabilities_response.rated_disabilities.each do |disability|
-        disability.maximum_rating_percentage = 10 if disability.diagnostic_code == tinnitus
+      return unless Flipper.enabled?(:disability_526_maximum_rating_api)
+
+      vro_client = VirtualRegionalOffice::Client.new
+
+      rated_disabilities = rated_disabilities_response.rated_disabilities
+      diagnostic_codes = rated_disabilities.map(&:diagnostic_code)
+      params = { diagnostic_codes: }
+
+      begin
+        response = vro_client.get_max_rating_for_diagnostic_codes(params)
+      rescue Common::Client::Errors::ClientError => e
+        Rails.logger.error "Get Max Ratings Failed  #{e.message}.", backtrace: e.backtrace
+      else
+        ratings_response = response.body
+        return if ratings_response.blank? || ratings_response['ratings'].empty?
+
+        rated_disabilities.each do |disability|
+          rating_item = ratings_response['ratings'].find do |rating|
+            rating['diagnostic_code'] == disability.diagnostic_code
+          end
+          disability.maximum_rating_percentage = rating_item['max_rating'] if rating_item.present?
+        end
```

Comment:
This method is getting a fair bit longer than Ruby style likes. You could break the VRO part out into a separate class method, or perhaps just the API call -- whichever makes sense and results in a good split.